### ISSUE TITLE
Overwrite default position for Number column

### ISSUE THREAD

**Issue by popovsergiy** [ThaDafinser/ZfcDatagrid#220]
I create Number column and try overwrite default text position from right to left but this has no affect. 
```php
$col = new Column\Select('quantity', 'p');
$col->setLabel('1C Quantity');
$col->setTranslationEnabled();
$col->setType(new Type\Number());
$col->addStyle(new Style\Align(Style\Align::$LEFT));
$grid->addColumn($col);
```
Renderer is jqGrid. 
http://screencast.com/t/DmaTiSQa7K

**Comment by ThaDafinser** [comment]
Right is applied here from the column type https://github.com/ThaDafinser/ZfcDatagrid/blob/master/src/ZfcDatagrid/Renderer/JqGrid/View/Helper/Columns.php#L85-L87

And the styling is done with the formatter.
https://github.com/ThaDafinser/ZfcDatagrid/blob/master/src/ZfcDatagrid/Renderer/JqGrid/View/Helper/Columns.php#L162

So it seems the align is more important then formatting for jqGrid.

Possible solution: Move the aligning to the formatter.

**Comment by popovsergiy** [comment]
I prefer PHP formatters since many JS formatters can break page during load.
This is not critical for my project at this moment and I will wait when this will fixed.

**Comment by ThaDafinser** [comment]
Should be closed by #228
